lib/karafka/helpers/class_matcher.rb

```diff
@@ -45,6 +45,13 @@ def match
       #   matcher.name #=> Super2Responder
       def name
         inflected = @klass.to_s.split('::').last.to_s
+        # We inject the from into the name just in case it is missing as in a case like that
```

Comment:
```
We inject the from into the name just in case it is missing as in a situation like that it would just sanitize the name without adding the "to" postfix. It could create cases when we want to build for example a responder to a consumer that does not have the "Consumer" postfix and would do nothing returning the same name. That would be bad as the matching classes shouldn't be matched to themselves.
```